<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPSClass</class>
 <widget class="QMainWindow" name="XPSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>工具栏</string>
    </property>
    <widget class="QPushButton" name="penButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/XPS/pen.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="eraseButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/XPS/eraser.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="colorButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 44, 115);
background-color: rgb(255, 34, 100);</string>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
    <widget class="QSlider" name="penSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>230</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>画笔/橡皮擦大小</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>861</width>
      <height>611</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="XImage" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>900</width>
       <height>609</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openaction"/>
    <addaction name="saveaction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openaction">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveaction">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XImage</class>
   <extends>QWidget</extends>
   <header>ximage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xps.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>penSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XPSClass</receiver>
   <slot>SetPen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorButton</sender>
   <signal>clicked()</signal>
   <receiver>XPSClass</receiver>
   <slot>SetColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraseButton</sender>
   <signal>clicked()</signal>
   <receiver>XPSClass</receiver>
   <slot>SetEraser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>penButton</sender>
   <signal>clicked()</signal>
   <receiver>XPSClass</receiver>
   <slot>SetPen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetPen()</slot>
  <slot>SetColor()</slot>
  <slot>SetEraser()</slot>
 </slots>
</ui>
